### PR TITLE
Added member card directive

### PR DESCRIPTION
**Pull request by Yuuki-chan**
To use this directive, the HTML would look like:

`<member-card data="data1"></member-card>`

Pass in an object for the data like this:

```
$scope.data1 = {
	userName: "some name 1",
	userId: "sdfjaalsdjfldjsf",
	title: "teacher",
	invitations: [1, 2, 3, 4, 5],
	skillSet: ['java', 'python', 'english', 'arts', 'math', 'algo', '1', 'dsjfksldjfk', 'ksdjfkls']
};
```
but `invitations` can be a list of any data structure (not necessarily int). Don't forget to include member-card.js in your HTML file.

The card looks like this:

member_card_screenshot

### CODE REVIEW

**Comment by vishalbollu** [comment]
are invitations ids for other teams?

**Comment by Yuuki-chan** [comment]
@vishalbollu The invitation objects can be the invitation data structures themselves or just a list of teams the person has been invited to.

File: app/directives/member-card.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+(function (angular) {
+    'use strict';
+
+    /*
+     * A template for member cards to be used throughout the website.
+     */
+    angular.module('teamform-member-app').directive('memberCard', function () {
```
Comment (vishalbollu) [inline app/directives/member-card.js]: the app is no longer called teamform-member-app. Its just called teamform

Comment (Yuuki-chan) [reply app/directives/member-card.js]: Oh...right, let me change that. Thanks for catching that.

File: app/directives/member-card.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+     * A template for member cards to be used throughout the website.
+     */
+    angular.module('teamform').directive('memberCard', function () {
+        return {
+            restrict: 'E',
+            replace: true,
+            scope: {
+                data: '=data'
+            },
+            templateUrl: 'directives/templates/member-card.html'
```
Comment (vishalbollu) [inline app/directives/member-card.js]: After the angular refactoring, this url may need to be 'app/directives/templates/member-card.html'

Have you tried using the directive with the refactored app?

**Comment by vishalbollu** [comment]
Also on a follow up note, do you plans to test this?

**Comment by Yuuki-chan** [comment]
@vishalbollu fixed the module name and the path error. Tested that this still works with the refactored code. Thanks for catching those.

Yeah, I can write my own tests once the tests are set up (are they?).

**Comment by vishalbollu** [comment]
LGTM

I think testing for directives can be looked at here:
https://docs.angularjs.org/guide/unit-testing#testing-directives

**Review by vishalbollu — APPROVED** [review]

**Pull request merged**